Source/VCGeneration/VCGen.cs

```diff
@@ -729,7 +732,13 @@ private void ConvertCFG2DAGStandard(Implementation impl, Dictionary<Block, List<
                 b = new LoopInitAssertCmd(c.tok, c.Expr, c);
               }
 
-              b.Attributes = c.Attributes;
+              b.Attributes = (QKeyValue)c.Attributes?.Clone();
+              if (Options.PrintVerificationCoverage) {
```

Comment:
I don't think `Print` is the right name of this option, since you should also be able to use it in a fully programmatic way in which the consumer never prints anything. Maybe the Dafny IDE presents it through some UI for example.